src/entities/transaction/model.ts

```diff
@@ -30,28 +28,25 @@ function hexToBytes(hex: string) {
 }
 
 export function mapFromDto(transaction: TransactionDto): Transaction {
-  const instructions =
-    transaction.c.payload.instructions.c?.map((i: string) => Instruction.fromBuffer(hexToBytes(i))) ?? [];
+  const instructions: Instruction[] = [];
+  // TypeError: Invalid argument type in ToBigInt operation (@iroha2_data-model.js:240)
+  //   transaction.c.payload.instructions.c?.map((i: string) => Instruction.fromBuffer(hexToBytes(i))) ?? [];
+  // probably incompatibility between the version of @iroha2/data-model package and Iroha itself
 
   return {
-    committed: transaction.t === 'Committed',
-    block_hash: transaction.c.block_hash,
-    block_height: transaction.c.block_height ?? 0,
-    hash: transaction.c.hash,
-    signatures: transaction.c.signatures,
+    committed: !transaction.rejection_reason,
+    block_hash: transaction.block_hash,
+    block_height: transaction.block_height,
+    hash: transaction.hash,
+    signatures: transaction.signatures,
     payload: {
-      ...transaction.c.payload,
+      ...transaction.payload,
+      creation_time: new Date(transaction.payload.creation_time),
       instructions,
     },
   };
 }
 
-export async function fetchList(params?: PaginationParams): Promise<Paginated<Transaction>> {
-  const res = await http.fetchTransactions(params);
-  const data = res.data.map(mapFromDto);
-
-  return {
-    pagination: res.pagination,
-    data,
-  };
+export async function fetchList(params?: PaginationParams): Promise<Paginated<TransactionDto>> {
```

Comment:
This function used to return a mapped `Transaction`, not a raw `TransationDto`. Why you changed it? Why it was the way it was, in your opinion?